profile/README.md

```diff
@@ -0,0 +1,60 @@
+# Dart
+
+## A client-optimized language for fast apps on any platform
+
+Dart is:
+
+  * **Optimized for UI**: Develop with a programming language specialized     
+    around the needs of user interface creation.
+
+  * **Productive**: Make changes iteratively: use hot reload to see the result
+    instantly in your running app.
+  
+  * **Fast on all platforms**: Compile to ARM & x64 machine code for mobile,
+    desktop, and backend. Or compile to JavaScript for the web.
+
+Dart's flexible compiler technology lets you run Dart code in different ways,
+depending on your target platform and goals:
+
+  * **Dart Native**: For programs targeting devices (mobile, desktop, server,
+    and more), Dart Native includes both a Dart VM with JIT (just-in-time)
+    compilation and an AOT (ahead-of-time) compiler for producing machine
+    code.
+  
+  * **Dart Web**: For programs targeting the web, Dart Web includes both a
+    development time compiler (dartdevc) and a production time compiler
+    (dart2js).  
+
+![Dart platforms illustration](../assets/Dart-platforms.svg)
```

Comment:
Somehow the markdown preview doesn't show this, but the link looks like it would work when landed.